tst/EnergyPlus/unit/CMakeLists.txt

```diff
@@ -51,7 +51,11 @@ set( test_src
   General.unit.cc
   GroundHeatExchangers.unit.cc
   HeatBalanceManager.unit.cc
+<<<<<<< HEAD
```

Comment:
This is the problem @RKStrand; some git merge conflict markers were left in the source after the merge.....  Looks like the resolution is to simply **keep** both unit.cc files listed here and delete the markers.  I'll do that now actually...